src/containers/LogFace/SelectLocationFilter/index.tsx

```
@@ -0,0 +1,153 @@
+/** select dropdown that allows user to filter by location,
+ * some constraints applied:
+ * - user can only filter for locations that they have been allowed access to,
+ * we determine this by using the locationHierarchy of the user returned from security authenticate
+ */
+import React, { useEffect, useState } from 'react';
+import { Fragment } from 'react';
+import { useTranslation } from 'react-i18next';
+import Select, { components } from 'react-select';
+import { TreeNode } from 'store/ducks/locationHierarchy/types';
+
+interface SelectLocationFilterProps {
+    userLocationTree?: TreeNode;
+    userLocationId: string;
+    disabled: boolean;
+    onLocationChange: (value: string) => void;
+}
+
+const defaultProps = {
```

We should avoid having default props. It makes things fail silently. If the prop is required, it should be passes to the component. https://kentcdodds.com/blog/prop-drilling